README.md

```diff
@@ -26,30 +26,27 @@ implementations are ill-suited for container environments:
   centralized logging when a syslog server is running, but with containers,
   simply logging to `stdout` or `stderr` is preferred.
 
-Finally, they are often very quiet, which makes the above issues difficult to
-understand or debug!
+Finally, they are often quiet, making these issues difficult to understand and debug!
 
-The list could go on, but the fundamental takeaway is this: unlike typical
-server cron implementations, Supercronic tries very hard to do exactly what you
-expect from running `cron` in a container:
+Supercronic's goal is to behave how you would expect `cron` running in a container to behave:
 
-- Your environment variables are available in jobs.
-- Job output is logged to `stdout` / `stderr`.
+- Your environment variables are available in jobs
```

Comment:
Periods or no periods are fine either way, just aiming consistency within the list.